### DISCUSSION TITLE
Future possible reddit downloader

### DISCUSSION THREAD

**Discussion by BlipRanger** [ArchiveBox/ArchiveBox#754] [Ideas]
Just wanted to make a quick mention of [BDfR](https://github.com/aliparlakci/bulk-downloader-for-reddit) as a cool project that might make for a good starting point for the unrolling of reddit comments/posts as mentioned in the roadmap. They currently support grabbing a variety of media types from the post as well as the comments/text in a separate (json) file. I've been working on an [addon](https://github.com/BlipRanger/bdfr-html) for it lately and I think it's a pretty great project with well-maintained code. If nothing else, they have really good examples of working with reddit data which could be useful! Just wanted to bring that to your attention!

**Comment by pirate** [comment]
👍 Good idea. Moving this discussion into a new feature request ticket ➡ https://github.com/ArchiveBox/ArchiveBox/issues/778
